outlier_detection.ipynb

```diff
@@ -0,0 +1,264 @@
+{
```

Comment:
I assume this notebook is not supposed to be in here? Will only review the other one in cifar10-outlier-detection/outlier_detection.ipynb

Reply:
@ulya-tkch Any update?